fjord/feedback/static/js/android_feedback.js

```diff
@@ -0,0 +1,66 @@
+(function($) {
+  'use strict';
+
+  // settings
+  var browserReviewUrl = 'https://play.google.com/store/apps/details?id=org.mozilla.CHANNEL#details-reviews';
+  var deviceReviewUrl = 'market://details?id=org.mozilla.CHANNEL';
+
+  // initialize
+  var reviewUrl, lastUrl = '';
+  var url = decodeURIComponent(window.location.href);
+
+
+  /*
+    Decide whether they're on Fx Android or not
+    (Fx Android's feedback prompt includes a utm param)
+  */
+  var fromDevice = /source=feedback-prompt/.test(url);
+  if (fromDevice) {
+    reviewUrl = deviceReviewUrl;
+    /* 
+      TODO: When bug 1237388 is resolved, remove #play-store here.
+      If url params are as assumed below, it'll Just Work(tm)!
+    */
+    $('#maybe-later, #no-thanks, #play-store').click(
+      function(e) {
+        e.preventDefault();
+        var feedbackEvent = document.createEvent('Events');
+        feedbackEvent.initEvent(this.getAttribute('data-event'), true, false);
+        this.dispatchEvent(feedbackEvent);
+      }
+    );
+  }
+  else {
+    reviewUrl = browserReviewUrl;
+    // if browsed here deliberately, no need for these
+    $('#maybe-later, #no-thanks').hide();
+  }
+
+  // decide what product they might review on play store
+  var version = $('body').attr('data-version');
+  var channel = $('body').attr('data-channel');
+  if (channel == 'beta') {
+    reviewUrl = reviewUrl.replace('CHANNEL', 'firefox_beta');
+  }
+  else if (channel != '' && /^(?!(beta|release)).*$/.test(channel)) {
```

Comment:
I generally prefer using things like `startsWith` instead of regexes when available, as I think it's clearer to read. But that is also totally a personal preference thing; feel free to leave this if you disagree.